x/wasm/ioutils/ioutil.go

```diff
@@ -3,17 +3,16 @@ package ioutils
 import (
 	"bytes"
 	"compress/gzip"
+	"fmt"
 	"io"
-
-	errorsmod "cosmossdk.io/errors"
-
-	"github.com/CosmWasm/wasmd/x/wasm/types"
 )
 
+var errLimit = fmt.Errorf("exceeds limit")
+
 // Uncompress expects a valid gzip source to unpack or fails. See IsGzip
 func Uncompress(gzipSrc []byte, limit int64) ([]byte, error) {
 	if int64(len(gzipSrc)) > limit {
-		return nil, types.ErrLimit.Wrapf("max %d bytes", limit)
+		return nil, fmt.Errorf("%s: max %d bytes", errLimit, limit)
```

Comment:
Here an others: wrapping the error via stdlib is fine. Please use `%w` for this. 
```suggestion
		return nil, fmt.Errorf("%w: max %d bytes", errLimit, limit)
```
Alternatively you can also use:
```suggestion
		return nil, errorsmod.Wrapf(errLimit, "max %d bytes", errLimit, limit)
```